### PR TITLE
CVE-2017-7465

### PR DESCRIPTION
**Pull request by sfowl**

### CODE REVIEW

File: 2017/7xxx/CVE-2017-7465.json [hunk 1]
```diff
@@ -1,18 +1,71 @@
 {
```
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7465.json]: [] is too short

**Comment by sfowl** [comment]
Fixed

File: 2017/7xxx/CVE-2017-7465.json (hunk 1 above)
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7465.json]: u'n/a' is not of type u'array'

File: 2017/7xxx/CVE-2017-7465.json [hunk 2]
```diff
@@ -1,18 +1,73 @@
...
```
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7465.json]: Expecting , delimiter: line 20 column 57 (char 632)

**Comment by cve-team** [comment]
Thanks for the submission, this will be published on cve.mitre.org shortly.

**Pull request merged**